google-cloud-storage/acceptance/storage/file_test.rb

```diff
@@ -247,6 +247,18 @@
     uploaded.metadata["title"].must_equal meta[:metadata][:title]
   end
 
+  it "should create and update storage_class" do
+    uploaded = bucket.create_file files[:logo][:path], "CloudLogo-storage_class.png", storage_class: :nearline
+
+    uploaded.storage_class.must_equal "NEARLINE"
+    uploaded.storage_class = :dra
+    uploaded.storage_class.must_equal "DURABLE_REDUCED_AVAILABILITY"
```

Comment:
What do you think about changing the target storage class? I'm just afraid that every once in while, people look to tests for best practices, and `DURABLE_REDUCED_AVAILABILITY` is one that it is recommended to [move *away* from](https://cloud.google.com/storage/docs/storage-classes#dra).